localstack/services/edge.py

```diff
@@ -133,10 +133,13 @@ def get_port_from_custom_rules(method, path, data, headers):
     if re.match(r'^/queue/[a-zA-Z0-9_-]+$', path):
         return config.PORT_SQS
 
+    if path == '/' and to_bytes('QueueName=') in data_bytes:
+        return config.PORT_SQS
+
     # TODO: move S3 public URLs to a separate port/endpoint, OR check ACLs here first
     stripped = path.strip('/')
     data_bytes = to_bytes(data or '')
```

Comment:
Move this above 136 to fix the linting error

Reply:
Good catch, thanks - fixed.